### PR TITLE
TVB-2356: Update tutorial 0

### PR DESCRIPTION
**Pull request by VinczeRobert**

### CODE REVIEW

**Comment by VinczeRobert** [comment]
I kind of managed to update the tutorial 0, but there are 2 problems. One is that I couldn't rerun the R and Matlab code (we need to do this to update the figures and maybe the code needs update to) because no online app could run the hdf5 related files. I spent a few hours on this and couldn't find a workaround yet. Some suggestions would be welcome. 

The second and smaller problem is that a few lines seem to have different indentations and I don't know why (at least when I look on github). I tried to create the documentation for this branch by running python build_step1.py $TVB-2356 from the command line but I had some problems with sphinx (yes, I installed the version we have in Dockerfile-build).

File: tvb_documentation/tutorials/tutorial_0_GettingStarted.rst [hunk 1]
```diff
@@ -4,187 +4,218 @@
...
 
 1. Select the connectivity you want to share.
 
 2. In the *metadata overlay*, go to the tab *Links*. You’ll see a list with all
-   your projects. 
+   your projects except the current one.
 
-3. Link this datatype (connectivity) with the project you’ll share (e.g.
-   *Session\_II\_ShareAndLink\_b*).
+3. Link this datatype (connectivity) with the project you’ll share.
```
Comment (popaula937) [inline tvb_documentation/tutorials/tutorial_0_GettingStarted.rst:168]: Explicitly link to SecondNewProject and at point 5 mention that the user should switch to SecondNewProject in order to see the linked dt

File: tvb_documentation/tutorials/tutorial_0_GettingStarted.rst [hunk 2]
```diff
@@ -4,187 +4,218 @@
...
-you linked from project *Session\_II\_ShareAndLink\_a*.
 
 ------------------------
 
-Export and Read
+Export and Read Datatype
 ---------------
 
-#. Go to *Project > Data structure*
+1. Select *ModelingEpilepsy* as your current project.
```
Comment (popaula937) [inline tvb_documentation/tutorials/tutorial_0_GettingStarted.rst:205]: Could be helpful to show a screenshot with the button to click in order to select a project

File: tvb_documentation/tutorials/tutorial_0_GettingStarted.rst [hunk 3]
```diff
@@ -4,187 +4,218 @@
...
 
-#. Click on |node_tr| from *TimeSeriesRegion - conn\_192*.
+4. From the overlay, *Export* tab, download only the time series h5 file by clicking on *TVB format* (h5).
 
-#. From the overlay, *Export* tab, download the data in TVB format (h5).
+5. If you wish to download not just the time series file, but the linked datatypes as well,
+choose the *TVB Format with links* option.
+
+.. figure:: figures/LinkAndShare_LinkedProject.png
+   :alt: Export with or without links
```
Comment (popaula937) [inline tvb_documentation/tutorials/tutorial_0_GettingStarted.rst:217]: This figure seems wrong here

File: tvb_documentation/tutorials/tutorial_0_GettingStarted.rst [hunk 4]
```diff
@@ -198,54 +229,44 @@ start the jupyter_notebook script.
...
-
-    In [8]: plt.ylabel('amplitude [au]')
-    Out[8]: <matplotlib.text.Text at 0x118e9a190>
-
-    In [9]: plt.title(f.attrs['TVB_Title'])
-    Out[9]: <matplotlib.text.Text at 0x118eb0ad0>
+           plt.xlabel('time [ms]')
+           plt.ylabel('amplitude [au]')
+           plt.title(f.attrs['TVB_Title'])
+    Out[6]: <matplotlib.text.Text at 0x118eb0ad0>
```
Comment (popaula937) [inline tvb_documentation/tutorials/tutorial_0_GettingStarted.rst:269]: I would leave this figure as before, to match the Matlab and R examples.

File: tvb_documentation/tutorials/tutorial_0_GettingStarted.rst [hunk 5]
```diff
@@ -4,189 +4,220 @@
...
- ModelingEpilepsy                411 MB 
-============================    ========   
-
-------------------------
+First things first. We are going to use *tvb-data* which is present within the distribution
+(at TVB_Distribution/tvb_data/Lib/site-packages/tvb-data)
+or it can be downloaded from `here <https://zenodo.org/record/4263723#.YL9x4jaA7t0>`_.
+Other than that, please follow
+`this link <https://www.thevirtualbrain.org/tvb/zwei/client-area/public?_ga=2.222814555.1504534223.1522695590-1355432300.1522695590#>`_
+to download the *ModelingEpilepsy.zip (3)* project.
```
Comment (popaula937) [inline tvb_documentation/tutorials/tutorial_0_GettingStarted.rst:12]: Could we keep directly ModelingEpilepsy.zip on ladeluke and rename the original as ModelingEpilepsy_old.zip?

File: tvb_documentation/tutorials/tutorial_0_GettingStarted.rst [hunk 6]
```diff
@@ -4,189 +4,220 @@
...
 other users and/or link data to other TVB projects.  After this tutorial you
-should be able to import a new dataset, export a project and share it with a colleague.
-
-For your convenience, all the data were already generated. You can download
-the LinkandShare_a.zip and LinkandShare_b.zip files in the `TVB sharing area
-<http://www.thevirtualbrain.org/tvb/zwei/client-area/public>`_. We’ll only go
-through the steps required to reproduce some simulations.  You can always start
-over, click along and/or try to change parameters.
+should be able to import new dataset, create, import, export and delete a project and
+share your data with among your projects or with a colleague.
```
Comment (popaula937) [inline tvb_documentation/tutorials/tutorial_0_GettingStarted.rst:20]: to import a new dataset

File: tvb_documentation/tutorials/tutorial_0_GettingStarted.rst [hunk 7]
```diff
@@ -4,189 +4,220 @@
...
-8. Logout from your account and login with the other user account.
+You should be able to see the *ModelingEpilepsy* project in your other account as well.
 
-You should be able to see the connectivity matrix (and other datatypes)
-you linked from project *Session\_II\_ShareAndLink\_a*.
 
 ------------------------
 
-Export and Read
+Export and Read Datatype
```
Comment (popaula937) [inline tvb_documentation/tutorials/tutorial_0_GettingStarted.rst:202]: a Datatype?

File: tvb_documentation/tutorials/tutorial_0_GettingStarted.rst [hunk 8]
```diff
@@ -4,189 +4,220 @@
...
+4. If you wish to import a project, on the right column, bottom corner,
+   click on *Import project structure*.
 
-In this exercise we will assume that multiple users are working with the
-same TVB installation.
+.. figure:: figures/Projects_Import.png
+   :alt: Click on Import project structure
+   :scale: 30%
 
+Click on *Upload*.
```
Comment (popaula937) [inline tvb_documentation/tutorials/tutorial_0_GettingStarted.rst:92]: would be more clear: Choose the ModelingEpilepsy.zip project and click on Upload?

**Review by popaula937 — COMMENTED** [review]
Looks good, few remarks and we can merge

**Pull request merged**